ui/src/dashboards/components/TableOptions.tsx

```diff
@@ -0,0 +1,205 @@
+import React, {PureComponent} from 'react'
+import {connect} from 'react-redux'
+import {bindActionCreators} from 'redux'
+
+import GraphOptionsCustomizeFields from 'src/dashboards/components/GraphOptionsCustomizeFields'
+import GraphOptionsFixFirstColumn from 'src/dashboards/components/GraphOptionsFixFirstColumn'
```

Comment:
Nit: "Freeze" might be better than "Fix" here. Excel/Google Spreadsheets/Numbers use that term, and it's further away from bugfix/hotfix.